tests/interfaces/test_tensorflow.py

```diff
@@ -575,13 +575,8 @@ def expand(self):
                 U3(p[0], p[1], p[2], wires=0)
                 qml.expval(qml.PauliX(0))
 
-            qtape = qtape.expand()
             res = execute([qtape], dev, **execute_kwargs)[0]
 
-            assert qtape.trainable_params == [1, 2, 3, 4]

```

Comment:
As the tape is expanded inside `execute`, `trainable_params` isn't set for the tape we pass in.